NEWS.md

```diff
@@ -51,6 +65,34 @@ The format is based on [Keep a Changelog]
 
 ### Fixed
 
+- When looking for menu.xml, go through all paths rather than just giving up
+  if there is no menu.xml in the first path searched? This makes it consistent
+  with how other config/theme files are handled. PR #1971
+- Fix memory leaks in theme.c and menu.c. PR #1971
+- Fix session-lock bugs related to keyboard focus. PR #1952
+  - Clear focused surface on lock
+  - Restore focused view on unlock
+  - Never allow multiple session locks
```

Comment:
The last one is what I introduced and fixed in the same release cycle so I think can be omitted.

Reply:
Done.

Assume you meant to just remove "Never allow multiple session locks" rather than all four lines relating to #1952